.travis.yml

```diff
@@ -7,13 +7,18 @@ go:
   - 1.6
 
 env:
-  - TOOLS_CMD=golang.org/x/tools/cmd
+  global:
+    - TOOLS_CMD=golang.org/x/tools/cmd
+    - PATH=$GOROOT/bin:$PATH
 
 install:
  - go get ${TOOLS_CMD}/vet
+ - go get ${TOOLS_CMD}/cover
+ - go get github.com/modocache/gover
+ - go get github.com/mattn/goveralls
 
 script:
- - sudo -E /bin/bash -c 'PATH=$GOROOT/bin:$PATH ./test'
```

Comment:
It dawned on me that removing this line was probably the root cause of the issues you were seeing (as we saw similar path related issues when setting these tests up).

Due to the way that Travis sets up your `GOPATH` / `GOROOT` we came up with that sudo line so that we could hand all of the `ENV` vars through to the test script.

Bringing this line back should allow you to run the `ginkgo` command in the test script.